examples/InformationPanel.main.tsx

```diff
@@ -48,35 +42,12 @@ export default () => {
           <Text variant='subheading'>Row {openRowIndex ?? 0}</Text>
         </InformationPanelHeader>
         <InformationPanelBody>
```

Comment:
Would a little spacing between the paragraphs be better? Because they currently look like sentences from the same paragraph.

<img width="454" alt="Screenshot 2023-12-20 at 10 20 07 AM" src="https://github.com/iTwin/iTwinUI/assets/45748283/6fd47239-cda6-40b4-9bb6-032365ea90e0">

Reply:
this should be controlled by CSS but i don't want to add unnecessary code in the example